### ISSUE TITLE
Archlinux depext for conf-python-3

### ISSUE THREAD

**Issue by Gbury** [ocaml/opam-repository#18540]
The depexts for `conf-python-3` on archlinux seems incorrect. From what I can tell, the `python` package now provides `python3` (and there is an explicit `python2` package, see https://wiki.archlinux.org/index.php/python ), but is simply named `python` and the `3` is in the version of the package.

In fact trying to `pacman -S python3` will install the `python` package (version `3.9.3-1` currently it seems), but crucially, I presumes that the `python3` package will not appear in the lists of installed packages that opam depexts looks into, but instead will be listed as simply the `python` package.

As can be seen below, although python3 is tecnically installed on my machine, opam complains about it not being installed.
```
$ yay -Qi python3
Name            : python
Version         : 3.9.3-1
Description     : Next generation of the python high-level scripting language
Architecture    : x86_64
URL             : https://www.python.org/
Licenses        : custom
Groups          : None
Provides        : python3
Depends On      : bzip2  expat  gdbm  libffi  libnsl  libxcrypt  openssl  zlib
Optional Deps   : python-setuptools [installed]
                  python-pip
                  sqlite [installed]
                  mpdecimal: for decimal
                  xz: for lzma [installed]
                  tk: for tkinter [installed]
Required By     : afl-utils  asciidoc  bpytop  gdb  gdb-common  glslang  gstreamer  iotop  perf  python-appdirs  python-cairo  python-distro  python-evdev  python-gobject  python-greenlet  python-idna
                  python-lxml  python-msgpack  python-ordered-set  python-pexpect  python-pillow  python-psutil  python-ptyprocess  python-pynvim  python-pyparsing  python-pyxdg  python-six  python-urllib3
                  python-yaml  reflector  steam  subdl  youtube-dl
Optional For    : botan  git  glib2  graphviz  gtk2  lensfun  libevent  libproxy  lilv  obs-studio  openexr  tdb  texlive-core  texlive-music  usbutils  util-linux  vim  vim-runtime
Conflicts With  : None
Replaces        : python3
Installed Size  : 79.29 MiB
Packager        : Felix Yan <felixonmars@archlinux.org>
Build Date      : Fri 09 Apr 2021 01:35:02 AM CEST
Install Date    : Wed 14 Apr 2021 07:27:35 AM CEST
Install Reason  : Installed as a dependency for another package
Install Script  : No
Validated By    : Signature

guigui@space-guigui ~ $ opam install conf-python-3
[ERROR] Package conf-python-3 depends on the unavailable system package 'python3'. You can use `--no-depexts' to attempt installation anyway.
```

I guess the solution would simply be replacing `python3` by `python` in the depext for archlinux ? (is it possible to try and add a lower bound on version of external packages ?)

**Comment by psafont** [comment]
> I guess the solution would simply be replacing python3 by python in the depext for archlinux

Yes, that's the solution, conf-python-2-7 already points to the correct arch package (python2)

**Comment by kit-ty-kate** [comment]
Sorry for having overlooked this issue before. This is a bug in opam as well, see the meta issue tracking this category of problem: https://github.com/ocaml/opam/issues/4759. We'll try to have them fix for opam 2.1.1
